File: SOLUTIONS/Problem01.java
import java.util.Scanner;

public class Problem01 {
    public static void main(String[] args) {
        String[] names = {"Bhin Bhin", "Atung", "Kaka", "Hodori", "Pan Pan", "Appu", "Lulu", "Orry", "Mei Mei"};
        Scanner scanner = new Scanner(System.in);
        try {
            int index = Integer.parseInt(scanner.nextLine());
            if (index > 9) {
                if (index % 9 == 0) {
                    index = 9;
                } else {
                    index = index % 9;
                }
            }
            index -= 1;
            System.out.println(names[index]);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: SOLUTIONS/SOLUTION#01/Problem 01.py
arr1 = ["Bhin Bhin", "Atung", "Kaka", "Hodori", "Pan Pan", "Appu", "Lulu", "Orry", "Mei Mei"];
arr2 = ["Haha", "Hihi", "Huhu", "Hoho"];

given = int(input())

given1 = given % len(arr1)
given2 = given % len(arr2)

if given1 == 0 :
    print (arr1[len(arr1)-1])
else:
    print (arr1[given1-1])
if given2 == 0 :
    print (arr2[len(arr2)-1])
else:
    print (arr2[given2-1])

